include/dxc/DxilContainer/DxilPipelineStateValidation.h

```diff
@@ -12,20 +12,19 @@
 #ifndef __DXIL_PIPELINE_STATE_VALIDATION__H__
 #define __DXIL_PIPELINE_STATE_VALIDATION__H__
 
+// Don't include assert.h here.
+// Since this header is included from multiple environments,
+// it is necessary to define assert before this header is included.
+// #include <assert.h>
```

Comment:
Shall we remove `// #include <assert.h>`?

Reply:
I think it's fine. It's the original comment and use `// #include <assert.h>` here probably can catch more attention.